README.md

```diff
@@ -114,45 +101,7 @@ To view the serial output you can use any terminal client of your choosing such
 
 You can find more information on the Mbed OS configuration tools and serial communication in Mbed OS in the related [related links section](#related-links).
 
-The output should contain the following block transmitted at the blinking LED frequency (actual values may vary depending on your target, build profile, and toolchain):
```

Comment:
@kegilbert - Did we intend to remove the sample output section below? Everything else looks good to me.

Reply:
I did, the sample output adds little in my opinion and is very commonly out of date as it is liable to change with each release and will change between toolchains and targets. Rather than have misleading/outdated logs I thought it better to remove completely.

@AnotherButler did you have additional thoughts on this? It's come up before when we talked about long term what we wanted in the READMEs.